app/forms/arp_payments_form.rb

```diff
@@ -0,0 +1,47 @@
+class ArpPaymentsForm < QuestionsForm
+  set_attributes_for :intake,
+                     :received_stimulus_payment,
+                     :eip1_amount_received,
```

Comment:
should add a numericality validator like on the other AGI / stimulus payment forms, e.g.
```
validates :primary_prior_year_agi_amount, gyr_numericality: true, if: :not_blank?
```

`gyr_numericality` mostly ensures the value is less than postgres max of `2**31 - 1` so jokers that come in an hammer out big numbers on their keyboard don't cause a 500 exception

Reply:
good call -- thanks.